src/engine/strat_engine/names.rs

```diff
@@ -52,6 +52,12 @@ impl KeyDescription {
     }
 }
 
+impl VolumeKeyKeyDescription {
+    pub fn to_system_string(&self) -> String {
```

Comment:
I think you might be better off creating a type for the result value of ```to_system_string``` as there are more functions than formerly that require the specially formatted system string as an argument.

Reply:
Working on implementing this, I believe there are many more places where we don't have any place for a type signature that use `.to_system_string()`. The end result of this is that we have more places where we call `.to_system_string().into_string()` which looks a bit redundant to me. Because we're often using this specifically in contexts that expect a String and less in function signatures that take a String, this appears to be more disruptive than helpful.